webanno/webanno
# Feature/1612 upgrade to bootstrap 4

**What's in the PR**
* Upgrade to Bootstrap 4 and SCSS

**How to test manually**
* Use every possible UI of the application and check if it still works ok.

Test the following pages:
- [x] Annotation view
- [x] Projects view
- [x] User view
- [x] Monitoring view
- [ ] Curation 

**Automatic testing**

**Documentation**
